test/env.spec.ts

```diff
@@ -0,0 +1,176 @@
+} from "../lib/index";
+import { DD_HANDLER_ENV_VAR } from "../lib/redirect";
+
+describe("applyEnvVariables", () => {
+  it("applies default values correctly", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "us-west-2",
+      },
+    });
+    const hello = new lambda.Function(stack, "HelloHandler", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const DatadogCDK = new Datadog(stack, "Datadog", {
+      forwarderARN: "forwarder-arn",
+    });
+    DatadogCDK.addLambdaFunctions([hello]);
+    expect(stack).toHaveResource("AWS::Lambda::Function", {
+      Environment: {
+        Variables: {
+          [DD_HANDLER_ENV_VAR]: "hello.handler",
+          [logForwardingEnvVar]: transportDefaults.flushMetricsToLogs.toString(),
+          [enableDDTracingEnvVar]: defaultEnvVar.enableDDTracing.toString(),
+          [injectLogContextEnvVar]: defaultEnvVar.injectLogContext.toString(),
+        },
+      },
+    });
+  });
+});
+
+describe("enableDDTracingEnvVar", () => {
+  it("disables Datadog tracing when set to false", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "us-west-2",
+      },
+    });
+    const hello = new lambda.Function(stack, "HelloHandler", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const DatadogCDK = new Datadog(stack, "Datadog", {
+      enableDDTracing: false,
+    });
+    DatadogCDK.addLambdaFunctions([hello]);
+    expect(stack).toHaveResource("AWS::Lambda::Function", {
+      Environment: {
+        Variables: {
+          [DD_HANDLER_ENV_VAR]: "hello.handler",
+          [logForwardingEnvVar]: "true",
+          [enableDDTracingEnvVar]: "false",
+          [injectLogContextEnvVar]: "true",
+        },
+      },
+    });
+  });
+  it("enables Datadog tracing by default if value is undefined", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "us-west-2",
+      },
+    });
+    const hello = new lambda.Function(stack, "HelloHandler", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const DatadogCDK = new Datadog(stack, "Datadog", {
+      forwarderARN: "forwarder-arn",
+    });
+    DatadogCDK.addLambdaFunctions([hello]);
+    expect(stack).toHaveResource("AWS::Lambda::Function", {
+      Environment: {
+        Variables: {
+          [DD_HANDLER_ENV_VAR]: "hello.handler",
+          [logForwardingEnvVar]: "true",
+          [enableDDTracingEnvVar]: "true",
+          [injectLogContextEnvVar]: "true",
+        },
+      },
+    });
+  });
+  it("throws error if Datadog Tracing is enabled but forwarder is not defined", () => {
```

Comment:
Again, we probably don't need this behavior.

Reply:
Removed the test associated with it 👍